Cargo.toml

```diff
@@ -9,10 +9,11 @@ crate-type = ["cdylib", "rlib"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-leptos = { version = "0.4.10", features = ["nightly"] }
-leptos_actix = { version = "0.4.10", optional = true }
-leptos_meta = { version = "0.4.10", features = ["nightly"] }
-leptos_router = { version = "0.4.10", features = ["nightly"] }
+leptos = { version = "0.5.0-rc3", features = ["nightly"] }
+leptos_actix = { version = "0.5.0-rc3", optional = true }
+leptos_meta = { version = "0.5.0-rc3", features = ["nightly"] }
+leptos_router = { version = "0.5.0-rc3", features = ["nightly"] }
+leptos_icons = { git="https://github.com/Carlosted/leptos-icons.git", features = ["macros", "ChBell", "ChSearch", "ChMenuHamburger"] }
```

Comment:
I swapped the icon dependency because the old one hasn't received updates in months, and the maintainer didn't respond to my PR to bump the leptos version.

This uses a git url because the version that supports 0.5.x isn't on crates.io yet.

Reply:
Gotcha, I see your PR there. That's unfortunate that its unmaintained. I'll ping someone in the leptos matrix chats.